packages/gatsby-transformer-sharp/src/extend-node-type.js

```diff
@@ -16,6 +16,11 @@ const {
 } = require(`gatsby-plugin-sharp`)
 
 const sharp = require(`sharp`)
+const {promisify} = require('util');
```

Comment:
This is a Node 8 only function. You can use https://www.npmjs.com/package/util.promisify